CHANGELOG.md

```diff
@@ -4,6 +4,21 @@ All notable changes to this project will be documented in this file, per [the Ke
 
```

Comment:
Not a related change but line 53 seems to be broken because of the opening of the `<code>` tag when viewing in a [rendered blob mode](https://github.com/10up/ads-txt/blob/d4a550263995099145b48ed295f170678868b884/CHANGELOG.md).

https://github.com/10up/ads-txt/blob/d4a550263995099145b48ed295f170678868b884/CHANGELOG.md?plain=1#L53